devsetup/standalone/hugepages.yaml

```diff
@@ -0,0 +1,13 @@
+---
+parameter_defaults:
+  NovaSchedulerEnabledFilters:
+    - ComputeFilter
+    - ComputeCapabilitiesFilter
+    - ImagePropertiesFilter
+    - ServerGroupAntiAffinityFilter
+    - ServerGroupAffinityFilter
+    - PciPassthroughFilter
+    - NUMATopologyFilter
+  ComputeParameters:
+    NovaReservedHugePages: ["node:0,size:2M,count:1"]
+    KernelArgs: "default_hugepagesz=2M hugepagesz=2M hugepages=1"
```

Comment:
Does it mean the deployment will have 1 piece of 2MB huge page? I don't think nova can really consume that for a VM as a VM's needs more than 2MB memory and the memory is either fully backed by huge pages or not.

Reply:
why 2mb and not 1gb? also agree with @gibi . One more question if we set NUMATopologyFilter shoudn't we also set some cpu pinning?

Reply:
whatever works in CI? I picked it randlomly and small

Reply:
I don't know, that's a good point. I followed the osp17.1 docs for numapages, and it suggests to use that filter w/o mentioning pinning

Reply:
We don't need cpu pinning to test huge pages.
@bogdando if you set up a flavor with page size 2M and then try to boot a cirros VM with 512MB request then having 1 piece of 2M pages on the host will mean that the VM scheduling will fail as it would require 256 pieces of 2M pages to fulflill the 512MB request

Reply:
the new version of the PR looks good now from this perspective.l